lib/decrypt.rb

```diff
@@ -3,10 +3,15 @@
 require './lib/offsets'
 require './lib/shift'
 
+encryption = ARGV[0]
+decryption = ARGV[1]
+key = ARGV[2]
+date = ARGV[3]
+
 enigma = Enigma.new
-message = File.read(ARGV[0])
-hash = enigma.decrypt(message, ARGV[2], ARGV[3])
-file = File.open(ARGV[1], 'w')
+encrypt_file = File.read(encryption)
+file = File.open(decryption, 'w')
+hash = enigma.decrypt(encrypt_file, key, date)
 file.puts hash[:decryption]
 
-p "Created '#{ARGV[1]}' with the key #{hash[:key]} and date #{hash[:date]}"
+p "Created '#{decryption}' with the key #{hash[:key]} and date #{hash[:date]}"
```

Comment:
Currently on your master, you do not have a variable 'decryption', so check on this.